### DISCUSSION TITLE
(Question) devices in the background

### DISCUSSION THREAD

**Discussion by Citro12** [seemoo-lab/AirGuard#154] [Q&A] [answered]
What do devices with a dark background mean compared to devices with a light background?
![Screenshot_2023-04-17-13-26-23-211_de seemoo at_tracking_detection release](https://user-images.githubusercontent.com/7021754/232471648-ae3c5292-c3a2-409e-b895-bde3357db7c6.jpg)

**Comment by Sn0wfreezeDev** [comment, marked as answer]
Dark background is used for devices which are not in tracking mode

**Reply by Sn0wfreezeDev** [reply]
In the newest version, this is now much easier to understand 😊
